core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -59,7 +59,7 @@ import scala.util.control.NoStackTrace
   * @param conf H2O configuration
   */
 class H2OContext private (@(transient @param @field) val sparkContext: SparkContext, @(transient @param @field) conf: H2OConf) extends Logging
-  with Serializable with SparkDataFrameConverter with SupportedRDDConverter with DatasetConverter
```

Comment:
You're right. This change doesn't do actually anything.

Why we need `ConverterUtils` to be `Serializable`
> We need to make `ConverterUtils` `Serializable` because inner class generated by scala to java conversion references instance of SparkDataFrameConverter, PrimiveRDDConverter and so on in outer scope.

I completely removed the need for the `Serializable` in  #87 by separating ConverterUtils and making it just helper objects instead of traits. If it's ok, I would close this PR and wait for #87 as extracting the change from the whole PR seems like a lengthy process. Is it ok with you ?